fmriprep_rodents/workflows/bold/base.py

```diff
@@ -807,7 +805,7 @@ def init_func_preproc_wf(bold_file):
                     ('std2anat_xfm', 'inputnode.std2anat_xfm')]),
                 (bold_bold_trans_wf if not multiecho else bold_t2s_wf, carpetplot_wf, [
                     ('outputnode.bold', 'inputnode.bold')]),
-                (bold_bold_trans_wf, carpetplot_wf, [
+                (bold_reference_wf, carpetplot_wf, [
```

Comment:
Are we sure of this change?

Reply:
Do you mean because the mask is generated from EPI, rather than a structural that is projected to the EPI space, or because the mask isn't from the realigned BOLD? The image I'm working on has good overlap, but I suppose to be safe,  could add an extra step where we project another mask into this space.